README.md

```diff
@@ -18,7 +18,52 @@ The Vulcan asset consumer is a long running job that allows to feed the assets
 in the [Vulcan API] into the [Graph Asset Inventory] by consuming the [Vulcan
 assets stream].
 
+## Test
+
+Start the testing infrastructure:
+
+```
+./_script/setup
+```
+
+Execute the tests:
+
+```
+go test -count 1 -p 1 -cover ./...
+```
```

Comment:
When I wrote this I thought that instead of mentioning `_script/test` it was better to tell the user what was happening under the hood. However, now I realize that it is actually simpler to tell them to use `_script/test` and forget about the internal peculiarities of the test suite. I changed it.